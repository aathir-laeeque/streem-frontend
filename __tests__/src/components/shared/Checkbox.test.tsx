import { Checkbox } from '#components';
import React from 'react';
import { create, ReactTestRenderer } from 'react-test-renderer';

describe('<Checkbox />', () => {
  let snapshot: ReactTestRenderer;

  beforeAll(() => {
    const component = <Checkbox onClick={() => jest.fn()} />;
    snapshot = create(component);
  });

  it('should render and match the snapshot', () => {
    expect(snapshot.toJSON()).toMatchSnapshot();
  });
});
