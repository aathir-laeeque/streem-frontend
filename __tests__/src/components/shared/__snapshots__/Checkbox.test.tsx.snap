// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkbox /> should render and match the snapshot 1`] = `
<div
  className="sc-fzqNJr jExKwZ checkbox-input"
>
  <label
    className="container"
  >
    <input
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <span
      className="checkmark"
    />
  </label>
</div>
`;
